test/format/t.c

```diff
@@ -284,47 +284,15 @@ startup(void)
 
 	/* Create or initialize the home and data-source directories. */
 	if ((ret = system(g.home_init)) != 0)
-		die(ret, "home directory initialization failed");
+		testutil_die(ret, "home directory initialization failed");
 
 	/* Open/truncate the logging file. */
 	if (g.logging != 0 && (g.logfp = fopen(g.home_log, "w")) == NULL)
-		die(errno, "fopen: %s", g.home_log);
+		testutil_die(errno, "fopen: %s", g.home_log);
 
 	/* Open/truncate the random number logging file. */
 	if ((g.rand_log = fopen(g.home_rand, g.replay ? "r" : "w")) == NULL)
-		die(errno, "%s", g.home_rand);
-}
-
-/*
- * die --
- *	Report an error and quit, dumping the configuration.
- */
-void
-die(int e, const char *fmt, ...)
-{
-	va_list ap;
-
-	if (fmt != NULL) {				/* Death message. */
-		fprintf(stderr, "%s: ", g.progname);
-		va_start(ap, fmt);
-		vfprintf(stderr, fmt, ap);
-		va_end(ap);
-		if (e != 0)
-			fprintf(stderr, ": %s", wiredtiger_strerror(e));
-		fprintf(stderr, "\n");
-	}
-
-	/* Flush/close any logging information. */
```

Comment:
The generic ```testutil_die``` has a lot less functionality than this one - we don't want to lose that.